crates/ruff/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -494,6 +559,34 @@ fn check_values(checker: &mut Checker, names: &Expr, values: &Expr) {
     }
 }
 
+fn find_duplicates(elts: &Vec<Expr>) -> Vec<usize> {
```

Comment:
flake8-pytest-style's implementation:

https://github.com/m-burst/flake8-pytest-style/blob/1b0feb85b4e189242a42725e8a2045f0880598d9/flake8_pytest_style/visitors/parametrize.py#L103

```python
    def _check_parametrize_duplicates(
        self, node: ast.AST, values: Optional[ast.AST]
    ) -> None:
        """Checks for PT014."""
        if not isinstance(values, (ast.List, ast.Tuple, ast.Set)):
            return

        for (i, element1), (j, element2) in itertools.combinations(
            enumerate(values.elts, start=1), 2
        ):
            if check_equivalent_nodes(element1, element2):
                self.error_from_node(
                    DuplicateParametrizeTestCases, node, indexes=(i, j)
                )
```

Reply:
Should we replicate the original implementation?

Reply:
The original behavior seems nice because we can easily tell where duplicates are.

Reply:
Implemented the original behavior, but fixed the error range.